app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -103,11 +103,7 @@ def self.hostname_required?
   end
 
   def self.default_blacklisted_event_names
-    %w(
-      ConfigurationSnapshotDeliveryCompleted
-      ConfigurationSnapshotDeliveryStarted
-      ConfigurationSnapshotDeliveryFailed
-    )
+    Settings.blacklisted_event_names
```

Comment:
@durandom - I had ```::Settings``` and it made no difference when executed.
I think it would be best to get this working on a small scale, i.e. on AWS, then we can think about moving it to the base class and updating the other providers in a separate PR.